src/helpers/indexedDB/quiz.ts

```diff
@@ -89,7 +89,9 @@ export const getQuizzes = async (userId: number): Promise<Object[]> => {
   return quizzes;
 };
 
-export const saveQuizzes = async (quizzes: Quiz[]) => {
+export const saveQuizzes = async (quizzes: Quiz[], userId: number) => {
+  await quizzesC.remove({ userId });
```

Comment:
Why are we deleting all the quizzes? Can't we delete one by some unique param?

Reply:
we are replacing all user quizzes with the latest quizzes data. 
user can add/edit/delete categories and. questions in existing quizzes.

Reply:
Can't we track with the id?
If we add to db first, then use the id from the db to store in the indexedDB?

Reply:
so you are saying I first find quizzesToUpdate, quizzesToRemove and quizzesToAdd. and updated those quizzes by ids.

what is the benefit of it ?
it seems more complicate and time taking process as it requires more db operations. instead we should delete user's all quizzes data and insert latest data

Reply:
Nevermind. This looks good.